src/core/filter-from-graph.ts

```diff
@@ -0,0 +1,98 @@
+import { DepGraph, DepGraphInternal, NodeInfo, PkgInfo } from './types';
+import { DepGraphBuilder } from './builder';
+import * as isEmpty from 'lodash.isempty';
```

Comment:
No! No lodash. Lodash bad.

Reply:
It doesn't make sense to say no to lodash when this repo is actually using it

Reply:
Of course it does. Just because we're using it doesn't mean we want to use more of it. We want to remove it entirely